src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    'chip version': version,
+    wheels: wheels,
+    x: 0,
+    y: 0,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${
+        this['chip version']}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.x}, y=${this.y}`;
+    },
+
+    goForward(value = 1) {
+      if (value > 0) {
+        this.y = this.y + value;
+      }
+
+      return this;
+    },
```

Comment:
add empty line between methods. Please fix all cases

```suggestion
    },
    
```